packages/react-composites/src/composites/CallComposite/pages/UnsupportedBrowser.tsx

```diff
@@ -26,6 +28,22 @@ export type UnsupportedBrowserPageProps = {
 export const UnsupportedBrowserPage = (props: UnsupportedBrowserPageProps): JSX.Element => {
   /* @conditional-compile-remove(unsupported-browser) */
   const { onTroubleshootingClick, environmentInfo } = props;
+  /* @conditional-compile-remove(unsupported-browser) */
+  const adapter = useAdapter();
+  /* @conditional-compile-remove(unsupported-browser) */
+  const unsupportedEnvironmentFeature = adapter.getState().features?.unsupportedEnvironment;
+  /* @conditional-compile-remove(unsupported-browser) */
+  const onContinueClick =
+    unsupportedEnvironmentFeature === true
+      ? undefined
+      : unsupportedEnvironmentFeature === false || undefined
```

Comment:
does `false || undefined` work? I hink this needs to be `unsupportedEnvironmentFeature === false || unsupportedEnvironmentFeature === undefined`